frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/webpack-env": "^1.15.2",
     "ansi-to-react": "^5.1.0",
+    "is-electron": "^2.2.0",
```

Comment:
Do we really need another dependency just for this?
The package hasn't been updated for 2 years and is fairly simple:
https://github.com/cheton/is-electron/blob/master/index.js

Maybe we can just pass it a `REACT_APP` var to determine this is an electron app?

Reply:
hmm, I don't like the approach of setting the REACT_APP to determine the electron app because than we will have to take care of so many env vars and yes as you mentioned checking whether isElectron is quite simple and I think I should just copy the helper method from https://github.com/cheton/is-electron/blob/master/index.js(coz its also a standard way to check electron environment)